scripts/qemu/run-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set -x
+set -e
+
```

Comment:
pls move `numactl -H` in here, or into `start_qemu.sh`

Reply:
done